Rettelse PR (#62)
MrLongbottom/P9-Rapport: sections/introduction.tex

\section{Introduction} 
%% Argue briefly for the relevance of the studied area/problem — start from general and end with your concrete problem.

Many news articles are produced every day and the need for searching the news is becoming a more prominent and difficult task.

> A general challenge for search engines, no matter the environment, is how to acquire relevant search results from a given corpus. 

Reviewer: Citation needed

While some engines use simple algorithms, such as finding all articles that include a given search term and sorting by date, this may not always give useful results.
Information retrieval within search engines apply many different techniques for either ranking or filtering documents\cite{google_pagerank2006}.

In order to rank or filter these documents, some information is needed to support the underlying ranking or filtering.

> A popular way to extract this information is through the use of topic modeling.

Reviewer: is it the information usefull for ranking or filtering or is it just usefull information?

Reviewer: citation needed

Topic modeling is a method within machine learning and \gls{NLP} which builds a model that can find abstract topics within a corpus of text documents.
A topic model assigns a distribution of topics to each document in a corpus, which can be used for different applications.
More specifically, we look into \gls{lda} since it is currently the most used topic model, and is applicable for many situations\cite{lda}.
\gls{lda} uses Dirichlet distributions to calculate the probability of a word belonging to a given topic and a document containing a given topic.

> Topic modeling does not rank documents, which is why an additional part is necessary if we want to search the documents.

Author: reference lda section?

> Ranking articles can be done in multiple ways, where PageRank is commonly used\cite{google_pagerank2006}.

Author: ```suggestion
Ranking articles can be done in multiple ways, where PageRank is commonly a used method\cite{google_pagerank2006}.
```

> PageRank\cite{pagerank_1999} is a ranking algorithm, which is based on the random walk process.

Reviewer: den her kan bliver kopieret nede i pr afsnitet

The principle is to let a number of surfers walk between nodes in a graph, where each surfer takes one step in each iteration.

> After an appropriate number of steps, the number of surfers on each node will be the ranking of the given node, where more surfers give a higher rank.

Reviewer: ```suggestion
After an appropriate number of steps, the number of surfers at each node will be the ranking of the given node, where more surfers give a higher rank.
```

> The edges in the graph can be based on many interactions, such as hyperlinks, similarities, genres, etc.

Author: reference PageRank section?

> In \cite{yang2009topic} \citeauthor{yang2009topic} investigate how to combine topic model \gls{lda}, \gls{lm}, and PageRank.

Reviewer: ```suggestion
In \citeauthor{yang2009topic} \cite{yang2009topic} investigate how to combine topic model \gls{lda}, \gls{lm}, and PageRank.
```

They also investigate a modified version of the PageRank algorithm which takes topic-level information into account.
They use a scientific article dataset where they manually label 200 articles from their dataset based on related queries. 

> In this paper, we want to expand the framework which \cite{yang2009topic} have provided.

Reviewer: ```suggestion
In this paper, we want to expand the framework which \citeauthor{yang2009topic} \cite{yang2009topic} have provided.
```

Author: ```suggestion
In this paper, we want to expand the framework which they have provided.
```

However, the data set which \cite{yang2009topic} uses is based on scientific articles, which includes citation to other similar articles.
This citation scheme does not necessarily extent to other document types, such as news articles.

> 
> In this paper, we want to investigate whether using a similarity measure based on the topics generated by a topic model can be used with PageRank.

Reviewer: ```suggestion

In this paper, we want to investigate whether using a similarity measure based on the topics generated by a topic model can be used with PageRank to improve \gls{ir} methods.
```

The combination of \gls{lda} and PageRank is interesting, since underlying document topics can yield more information than usually given by a single document.

> These topics can be seen as a set of documents sharing a specific topic. 

Author: ```suggestion
These topics can be seen as a set of words shared between documents. 
```

When evaluating search engines, manual labeling of documents is usually used to measure the performance of models\cite{yang2009topic}\cite{Tang2008}.
Since the topics, that are created by the \gls{lda}, are distributions of topics in documents, we can use these to evaluate a search algorithm's retrieval performance. 

> 

Reviewer: meget brat overgang

Reviewer: we need something about the evaluation of queries

Our goal is thus to improve news article search by using a combination of \gls{lda}, \gls{lm}, and PageRank.

> For our experiments we use the Nordjyske database of news articles as our dataset.

Author: ```suggestion
For our experiments we use the Nordjyske database of news articles as our dataset.

```
breakline to divide up goals.

> We also want to investigate whether an automatic evaluation measure can be used to evaluate a search engine.

Author: ```suggestion
We also want to investigate how an automatic evaluation measure can be used to evaluate \gls{ir] methods.
```

> Here we want to look into whether it makes sense to make an evaluation metric based on whether search results are in a given topic.

Author: ```suggestion
Here we want to put focus on the \gls{ir} method understanding the underlying topics expressed in a query.
```
```suggestion
Here we want to put focus on the \gls{ir} method understanding the underlying topics expressed in a query.

```
again, breakline to separate goals.

> Finally, since the news article dataset does not include connections between the articles, how PageRank can be run on this needs to be investigated.

Author: ```suggestion
Finally, since our news article dataset does not include connections between the articles, we want to investigate how we can build an adjacency matrix based on the documents themselves, to be able to run PageRank.
```

With these goals we look at the following questions:

\begin{quote}
	\emph{How can information retrieval techniques be evaluated based on search queries?}
	\begin{itemize}
		\item \emph{How can queries be generated for a dataset?}

> 		\item \emph{How can evaluation be done in a way that favors abstraction, rather than word frequency?}

Reviewer: er vi sikre på at vi vil gå mod word freq?

	\end{itemize}
\end{quote}
\vspace{0.1 cm}

\begin{quote}

> 	\emph{How can PageRank be used on a document dataset with no connections between documents?}

Reviewer: ```suggestion
	\emph{Can PageRank be used on a document dataset with no explicit connection to improve Information Retriveal methods? }
```

> With these goals we look at the following questions:
> 
> \begin{quote}
> 	\emph{How can information retrieval techniques be evaluated based on search queries?}
> 	\begin{itemize}
> 		\item \emph{How can queries be generated for a dataset?}
> 		\item \emph{How can evaluation be done in a way that favors abstraction, rather than word frequency?}
> 	\end{itemize}
> \end{quote}
> \vspace{0.1 cm}
> 
> \begin{quote}
> 	\emph{How can PageRank be used on a document dataset with no connections between documents?}
> \end{quote}

Author: Jeg vil gerne lave dette til noget mere formal som vi kan referere, og mere til goals end spørgsmål.
Derudover kan jeg ikke så godt lide det første spørgsmål da det virker for generelt, som om vi har tænkt os at undersøge en masse evaluation metrics i dette paper.
```suggestion
These goals are expressed in \todo{ref goals}.

\begin{quote}
	\begin{itemize}
		\item \emph{How can we generate queries for a dataset to use for \gls{ir}?}
		\item \emph{How can evaluate \gls{ir} methods in a way that favors abstraction, rather than word frequency?}
	        \item \emph{How can we contruct an adjacency matrix based on a dataset with no explitcit connections between documents, to be able to use PageRank ?}
	\end{itemize}
\end{quote}
\todo[inline]{make this into its own goal class that is referenceable like with definitions.}
```
btw, ved godt at det var mig der tidligere ville fjerne 'explicit'. Det lyder bare godt.
